### DISCUSSION TITLE
Welcome to UDOIT Discussions!

### DISCUSSION THREAD

**Discussion by bagofarms** [ucfcdl/UDOIT#665] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by jonrussellsoftwaretester** [comment]
Nice app.
